### ISSUE TITLE
g4s simulation missing a charging station

### ISSUE THREAD

**Issue by hawesie** [strands-project/strands_morse#117]
@cburbridge where is it going to go? @mudrole1 please can you add it when Chris tells you where?

**Comment by hawesie** [comment]
It's difficult to do long-term testing without a charger as the battery on the robot runs down. @kunzel is there a way to stop the battery level decreasing in simulation?

**Comment by kunzel** [comment]
As a quick fix you can set the robot's discharging rate to 0.

rosed strands_morse scitosA5.py
edit this -> strands_morse/strands_sim/src/strands_sim/builder/robots/scitosA5.py

[..]
 self.battery.properties(DischargingRate=0.00)
[..]

**Comment by mudrole1** [comment]
I can add this but I havent heared where :)

**Comment by hawesie** [comment]
pinging @cburbridge

**Comment by adegroote** [comment]
Btw, Morse 1.3 has a ChargingZone feature, which can allow to do that properly.

**Comment by hawesie** [comment]
Excellent news. @kunzel does this mean we can drop our fork?

**Comment by kunzel** [comment]
Yes, I think we should do that. The only two things that require our version at the moment are the battery charging and scene generation. But when we are going to re-structure our strands-morse package after the review, we should also migrate these things.
